### DISCUSSION TITLE
Avatar shapes inconsistent across layout— not really an "issue" but... requesting feedback

### DISCUSSION THREAD

**Discussion by omicreativedev** [deso-protocol/dips#42] [Medium CIPs]
I'm super excited about all the UX tweaks and how modernized things are starting to look. 🥰
@carsenk @AdityaChaudhary0005 @harpreet9629 and others are doing wonderful things in way of making the frontend feel more current with today's interface trends. That said, I noticed on a live build that there are some inconsistencies with the avatar shapes.

- The new account switcher has a circle avatar
- Most of the site uses a square avatar with rounded corners
- The main profile pic on each profile is firmly square

After all the work that has been done, I don't want to take it upon myself to PR against others work (especially recent work.)

I think we should come to consensus about one style for all the avatars.

Asking for community feedback about which you'd vote for:

- Circle
- Rounded Corner Square
- Square

Thank you.

**Comment by jsonpreet** [comment]
2nd option.

**Comment by tijno** [comment]
Is this discussion better placed as CIPs discussions?

**Comment by lazynina** [comment]
i think #2 is the answer

**Reply by tijno** [reply]
+1 for Rounded Corner Square

**Comment by AikonMishima** [comment]
I vote for #2 as well.
